addons/account/views/account_move_views.xml

```diff
@@ -987,13 +987,17 @@
                                            class="mr4"
                                            options="{'no_create': True, 'no_open': True}"
                                            readonly="posted_before"/>
+
+                                    <span class="o_form_label mx-3 oe_edit_only"
+                                          groups="account.group_account_readonly"
+                                          invisible="move_type == 'entry'">
+                                        <span groups="base.group_multi_currency">in </span>
+                                    </span>
```

Comment:
It's better, but they're still not really aligned
![image](https://github.com/odoo/odoo/assets/24565125/33cf33fd-b33a-4673-966c-5ab0ee85daee)

In 17.0:
![image](https://github.com/odoo/odoo/assets/24565125/04429e28-20d5-45cf-9302-fcdca4a99dd5)

Can't we do better ?

Reply:
You're right, I saw that Payment Terms and USD would be aligned, but did not check further.
Should be better now, removed the margin